defaults/main.yml

```diff
@@ -13,23 +13,23 @@ midonet_gateway_external_nic: '{{ ansible_default_ipv4.interface }}'
 midonet_gateway_create_external_network: True
 midonet_gateway_external_network_name: 'ext-net'
 midonet_gateway_external_subnet_name: 'ext-subnet'
-midonet_gateway_external_subnet_pool_start: '{{ midonet_gateway_fip_net  | ipaddr("10") | ipaddr("address") }}'
-midonet_gateway_external_subnet_pool_end: '{{ midonet_gateway_fip_net | ipaddr("254") | ipaddr("address") }}'
-midonet_gateway_external_subnet_gw: '{{ midonet_gateway_fip_net  | ipaddr("1") | ipaddr("address") }}'
+midonet_gateway_external_subnet_pool_start: '{{ midonet_gateway_external_network_cidr  | ipaddr("10") | ipaddr("address") }}'
+midonet_gateway_external_subnet_pool_end: '{{ midonet_gateway_external_network_cidr | ipaddr("254") | ipaddr("address") }}'
+midonet_gateway_external_subnet_gw: '{{ midonet_gateway_external_network_cidr  | ipaddr("1") | ipaddr("address") }}'
 midonet_gateway_edge_router_name: 'edge-router'
 midonet_gateway_edge_network_name: 'net-edge1-gw1'
 midonet_gateway_edge_subnet_name: 'subnet-edge1-gw1'
 midonet_gateway_edge_port_name: 'port-edge1-gw1'
-midonet_gateway_fip_net: '200.200.200.0/24'
+midonet_gateway_external_network_cidr: '200.200.200.0/24'
```

Comment:
Maybe changing this actually-allocated network (owned by Embratel, a Brazilian telco)
with a private network as well? The MidoNet quick start guide uses TEST-NET,
TEST-NET-2, TEST-NET-3 as an example:
> * One floating IP network
>   * 192.0.2.0/24
> * Two MidoNet gateway nodes
>   * gateway1, connecting to bgp1 via eth1
>   * gateway2, connecting to bgp2 via eth1
> * Two remote BGP peers
>   * bgp1, 198.51.100.1, AS 64513
>   * bgp2, 203.0.113.1, AS 64514
> * Corresponding MidoNet BGP peers
>   * 198.51.100.2, AS 64512
>   * 203.0.113.2, AS 64512